Makefile

```diff
@@ -24,14 +24,23 @@ GO_LD_EXTRAFLAGS :=-X k8s.io/component-base/version.gitMajor="1" \
                    -X k8s.io/client-go/pkg/version.buildDate="$(shell date -u +'%Y-%m-%dT%H:%M:%SZ')" \
                    -X k8s.io/client-go/pkg/version.gitTreeState="$(SOURCE_GIT_TREE_STATE)"
 
-GO_BUILD_FLAGS :=-tags 'json1'
+LIBDM_BUILD_TAG = $(shell hack/libdm_tag.sh)
+LIBSUBID_BUILD_TAG = $(shell hack/libsubid_tag.sh)
+BTRFS_BUILD_TAG = $(shell hack/btrfs_tag.sh) $(shell hack/btrfs_installed_tag.sh)
+
+ifeq ($(DISABLE_CGO), 1)
+	override BTRFS_BUILD_TAG = exclude_graphdriver_devicemapper exclude_graphdriver_btrfs containers_image_openpgp
+endif
+
+GO_BUILD_FLAGS = -tags "json1
+GO_BUILD_FLAGS += $(BTRFS_BUILD_TAG) $(LIBDM_BUILD_TAG) $(LIBSUBID_BUILD_TAG)"
```

Comment:
The quotes looks odd here. There is just one `"` on every line.

Reply:
I squashed into a single line to avoid this.